### PR TITLE
Postfix Evaluation in C

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
Fixes # #1552

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
-  Implemented Postfix Evaluation in C
- 

#### Languages Used:
-  C
- 

#### Files Added:
- Postfix_Evaluation.c
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@somya-kapoor Squashed all commits to a single commit. Created a new pull request after squashing. Please review it.

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+int evaluate(int op1, int op2, char c)
+{
+    if(c == '+')
+       return op1 + op2;
+    else if(c == '-')
+       return op1 - op2;
+    else if(c == '*')
+       return op1 * op2;
+    else if(c == '/')
+       return op1 / op2;
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: indentation is incorrect

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+void print(struct stack s)
+{
+    int i;
+    for(i = 0 ; i <= s.Top ; i++)
+       printf("%d\t", s.A[i]);
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: correct the indentation

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    else if(c == '*')
+       return op1 * op2;
+    else if(c == '/')
+       return op1 / op2;
+}
+
+void print(struct stack s)
+{
+    int i;
+    for(i = 0 ; i <= s.Top ; i++)
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: remove the space before ;

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+int pop(struct stack *ps)
+{
+    int x;
+    if(ps -> Top == -1)
+       printf("Stack is empty!Element cant be deleted\n");
+    else
+    {
+       x = ps -> A[ps -> Top];
+       ps -> Top = ps -> Top - 1;
+       return x;
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: correct the indentation

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+            op1 = pop(&s);
+            op2 = pop(&s);
+            optr = s.A[i];
+            x = evaluate(op2, op1, optr);
+            push(&s, x);
+         }
+      }
+
+      printf("The result is :  ");
+      print(s);
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: correct the indentation

File: Postfix_Evaluation/Postfix _Evaluation.c [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+      printf("\n");
```
Comment (asha15) [inline Postfix_Evaluation/Postfix _Evaluation.c]: indentation

**Comment by HariniJeyaraman** [comment]
@asha15  sorry for the confusion. Postfix evaluation in C has already been merged by #1560 . I created another PR by mistake while sqaushing the commits.

**Comment by asha15** [comment]
Shall we close this ?

**Pull request closed without merging**